main/maven/test/resources/expected/--base-module/build.mill

```diff
@@ -0,0 +1,31 @@
+package build
```

Comment:
Let's compare the `scalafmt`-ed output with the `scalafmt`-ed expected build files, with an assertion to check that the expected build files are `scalafmt`-ed. That will make editing them much less fragile than now and make them much easier to read and understand